packages/react/src/components/AccountOverview/AccountOverview.stories.mdx

```diff
@@ -0,0 +1,160 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import AccountOverview from './AccountOverview.tsx';
+import Button from '../Button';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
+import Typography from '../Typography';
+
```

Comment:
Shall we remove this line?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/58/commits/4973d88a31c62a594c87052bf927a6d40ed14a9f